Remove onRedirectNavigate from endSessionRequest (#8066)
AzureAD/microsoft-authentication-library-for-js: lib/msal-browser/docs/logout.md

});
```

> ### Skipping the server sign-out

Reviewer: We don't need to remove this section, just update it to use the PCA config option instead

})

// Automatically on page load
msal.logoutRedirect({
    onRedirectNavigate: () => {
        // Return false to stop navigation after local logout
        return false;
    }
});
msal.logoutRedirect();
```

> Now when a user logouts out of another application, your application's front-channel logout url will be loaded in a hidden iframe, and MSAL.js will clear its cache to complete single-sign out.

Reviewer: Why was this removed?

Author: Sorry, I got a bit confused when editing the documentation, this should all be fixed now.

> Now when a user logouts out of another application, your application's front-channel logout url will be loaded, and MSAL.js will clear its cache to complete single-sign out.

Reviewer 2: I thought front channel logout is still applicable in an iframe, when the storage partitioning is not enabled. 

Another note: May be we will benefit from adding reference to https://learn.microsoft.com/en-us/entra/identity-platform/reference-third-party-cookies-spas#limitations-on-front-channel-logout-without-third-party-cookies or something similar.

Author: Added that back in. Included reference to that link as well.

```typescript
const msal = new PublicClientApplication({
    auth: {
        clientId: "my-client-id",
        onRedirectNavigate: (url) => {
            return false;
        }
    },
    system: {
        allowRedirectInIframe: true
    }
})

// Automatically on page load
msal.logoutRedirect();
```

> Please note that front-channel logouts are not always supported across browsers. Chromium enabled [Storage Partitioning](https://privacysandbox.google.com/cookies/storage-partitioning) and Firefox supports [similar standard](https://developer.mozilla.org/en-US/docs/Web/Privacy/Guides/State_Partitioning) limiting applications to execute front-channel logout. For official Entra documentation on this topic,  see [here](https://learn.microsoft.com/en-us/entra/identity-platform/reference-third-party-cookies-spas#limitations-on-front-channel-logout-without-third-party-cookies).

Reviewer 2: ```suggestion
Please note that front-channel logout is not always supported across browsers. Chromium enabled [Storage Partitioning](https://privacysandbox.google.com/cookies/storage-partitioning) and Firefox supports [similar standard](https://developer.mozilla.org/en-US/docs/Web/Privacy/Guides/State_Partitioning) limiting applications to execute front-channel logout. For official Entra documentation on this topic,  see [here](https://learn.microsoft.com/en-us/entra/identity-platform/reference-third-party-cookies-spas#limitations-on-front-channel-logout-without-third-party-cookies).
```

msal.logoutRedirect();
```

> There are some limitations to front-channel logouts. For more information, see [here](https://learn.microsoft.com/en-us/entra/identity-platform/reference-third-party-cookies-spas#limitations-on-front-channel-logout-without-third-party-cookies).

Reviewer 2: ```suggestion
Please note that front-channel logouts are not always supported across browsers. Chromium enabled [Storage Partitioning](https://privacysandbox.google.com/cookies/storage-partitioning) and Firefox supports [similar standard](https://developer.mozilla.org/en-US/docs/Web/Privacy/Guides/State_Partitioning) limiting applications to execute front-channel logout. For official Entra documentation on this topic,  see [here](https://learn.microsoft.com/en-us/entra/identity-platform/reference-third-party-cookies-spas#limitations-on-front-channel-logout-without-third-party-cookies).
```
